### ISSUE TITLE
mergePairs failed for example data

### ISSUE THREAD

**Issue by DaliangNing** [benjjneb/dada2#590]
mergers <- mergePairs(dadaFs, derepFs, dadaRs, derepRs, verbose=TRUE)
0 paired-reads (in 0 unique pairings) successfully merged out of 7605 (in 421 pairings) input.
0 paired-reads (in 0 unique pairings) successfully merged out of 5692 (in 379 pairings) input.

First time use DADA2. Installed v1.8.0 from bioconductor.
source("https://bioconductor.org/biocLite.R")
biocLite("dada2")

downloaded example data via the "example data" link on tutorial page. use four files of two samples for test.
F3D0_S188_L001_R1_001.fastq
F3D0_S188_L001_R2_001.fastq
F3D1_S189_L001_R1_001.fastq
F3D1_S189_L001_R2_001.fastq

use exact R scripts on the tutorial page (set all multithread=FALSE, since using windows).
Other steps are all good, but mergePaires returned "0 paired-reads".
On another computer (Windows), install DADA2 v1.6.0, got the same error.
Any idea or solution for this problem?

**Comment by benjjneb** [comment]
That is strange. You're sure no other commands were altered from the tutorial?
What is the output of `dadaFs[[1]]` when you run it just prior to the `mergePairs` command?
And also `dadaRs[[1]]`?

**Comment by glucksfall** [comment]
I agree with @benjjneb 
I employed those files and the tutorial at dada2 website as an example to teach students and worked fine. 
I attached the R script that analyzes those data files and the pdf. Beware that the script has multithread=TRUE because I use Ubuntu. Also, change the path pointing to your folder. I used dada2 v1.8.0.
[DADA2-tutorial-DataSet2.zip](https://github.com/benjjneb/dada2/files/2493707/DADA2-tutorial-DataSet2.zip)

Anyway, I'm having the same issue but with own data. I could merge from 0 to around 30. (```27 paired-reads (in 5 unique pairings) successfully merged out of 79079 (in 30221 pairings) input```). Could be that the forward and/or reverse sequences are inverted? or dada2 merge function know that the reverse sequence should be inverted before merging?

Thank you @benjjneb 
Best regards

PS @DaliangNing: If you want to use R inside a jupyter notebook, follow the instruction here https://irkernel.github.io/installation/. I could share with you the notebook that produced the pdf attached if that help you further to analyze the example data.
